mod/01_rating/spec/set_pattern/metric_type_plus_right_spec.rb

```diff
@@ -0,0 +1,20 @@
+# encoding: UTF-8
+
+describe Card::MetricTypePlusRightSet do
+  it 'works' do
+    Card::Auth.as_bot do
+      card = Card.create! name: 'Designer+MetricName', type_id: Card::MetricID,
+        subcards: {
+          '+*metric type' => {
+            content: 'wiki_rating',
+            type_id: Card::PhraseID
+          },
+          '+test' => 'Some content'
+        }
+
+        binding.pry
+      ca = Card['Designer+MetricName+test']
+      expect(ca.set_names).to include("Designer+MetricName+test+*metric type plus right")
```

Comment:
Line is too long. [89/80]<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.